### PR TITLE
Followup edits to PR 2426

### PR DESCRIPTION
**Pull request by bfallonf**
#2426

@bparees Can I get an ack I didn't change any meaning?

Thanks!

### CODE REVIEW

**Comment by bfallonf** [comment]
@ahardin-rh @tpoitras @tnguyen-rh Could probably do with a peer review as well.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../dev_guide/builds.adoc[Builds]
|Added dev_guide/builds.adoc#build-resources[Build Resources] section.
%

**Comment by ahardin-rh** [comment]
@bfallonf LGTM! :star2:

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -2028,16 +2028,13 @@ manually)
 [[build-resources]]
 == Build Resources
 
-A build is completed by a pod that consumes resources (memory and CPU) on a
-node. By default, pods consume unbounded node resources. However, if a project
-specifies default container limits, then pods consume resources up to those
-limits.
+When a pod completes a build, unbound resources, such as memory and CPU, are
```
Comment (bparees) [inline dev_guide/builds.adoc]: I think the original phrasing of the first sentence might be better here.  not all pods execute builds, so saying "when a pod completes a build" implies that one of the steps of running a pod is for it to complete a build, which is not true.

**Comment by bparees** [comment]
just one comment.

**Comment by bfallonf** [comment]
@bparees I'm not sure if I completely get the difference, but I changed it to look more like the original sentence. WDYT?

**Comment by bparees** [comment]
lgtm

**Comment by bfallonf** [comment]
@bparees cool, thanks!

@ahardin-rh has given the thumbs up (thanks!), so I'll merge this.

**Pull request merged**